### ISSUE TITLE
Slicer stalls when bridge settings enabled

### ISSUE THREAD

**Issue by kvnper** [smartavionics/Cura#51]
Hi SA,

I ran into an issue slicing a model when 'Enable Bridge Settings' is enabled. When enabled, the slicing stalls and does not proceed past half way. When disabled it slices swiftly. I haven't tried leaving it more than 10 minutes but usually, in regular cura, this setting does not increase slice time and paired with the fact that you modified the bridging code, I thought that it might be an issue with that. The model is fairly small, and maybe even simple, too.

I've attached the 3mf file (had to rename the extension to attach it here).

[CFFFP_octofinger (1).3mf.txt](https://github.com/smartavionics/Cura/files/4317495/CFFFP_octofinger.1.3mf.txt)

**Comment by kvnper** [comment]
I forgot to mention that it slices fine in cura 4.5 with the setting both enabled and disabled.

**Comment by smartavionics** [comment]
With my latest build it doesn't stall. Can you please provide the stderr.log and cura.log files for when it fails for you?

![Screenshot_2020-03-11_10-38-02](https://user-images.githubusercontent.com/585618/76408142-689a5880-6384-11ea-8b8f-a3c32c1b445c.png)


There's something I don't understand though which is the fact that the cross pieces are not being treated as bridges so I need to investigate why that is occurring.

**Comment by kvnper** [comment]
Ok I thought I had the latest build, but I'll try installing it again.

I'll get back to you with the logs shortly if I still have the same issue.

**Comment by kvnper** [comment]
No luck with the latest build 20200306

Log attached. I can see an error there related to 'bridge_extend_floating_walls'

[cura.log](https://github.com/smartavionics/Cura/files/4317784/cura.log)

**Comment by kvnper** [comment]
I should really get some sleep, I keep missing things! stderr.log attached.

[stderr.log](https://github.com/smartavionics/Cura/files/4317789/stderr.log)

**Comment by smartavionics** [comment]
I think that can be cured bu removing the cura cache files. On Linux they are to be found at ~/.cache/cura/master/. I don't know about Windows and I don't have a Windows machine running at the moment but I guess they would be under AppData ? AppData/Roaming?

**Comment by kvnper** [comment]
I deleted everything in that folder:
![image](https://user-images.githubusercontent.com/47488385/76412106-26563300-63e7-11ea-9c64-b373d0f71175.png)

Presumably it gathered the settings from my 4.5 install. The thing is this folder stores the settings for custom printers, deleting anything here deletes any custom printers, print profiles, configurations, and settings.

Even so, I backed up and deleted every install in /Roaming/cura and I'm still having the slicing issue.

I notice I'm not getting the "Bridge Extend Floating Walls" setting, even after trying to show all settings.

![image](https://user-images.githubusercontent.com/47488385/76412267-70d7af80-63e7-11ea-987c-145e6b74f4e4.png)
